Estimation/Simulation/es_esoq2.m

```diff
@@ -0,0 +1,52 @@
+function q_bi = es_esoq2(sm_output,es_const)
+%This is the main function to run the QUEST-2
+%   Input:
+%       sm_input: input file from Star Matching
+%       es_const: file containing the constants required.
+%       
+%   Output:
+%       q_bi: final estimated quaternion
+
+%% Code
+
+%extracting different variables form the star matching output
+st_N_Match = sm_output.N;
```

Comment:
@shashanksira13 
It would be better if you changed the nomenclature from 'st_' since we will not be using it anymore.
Ensure this across all of your code.